content/kubermatic/main/architecture/known-issues/_index.en.md

```diff
@@ -9,67 +9,68 @@ weight = 25
 ```
 
-API server logs
+This problem arises when Cilium is used as CNI for the underlying seed cluster.
 
-```
-2022-05-26T11:46:11.269134597Z stderr F E0526 11:46:11.267368       1 authentication.go:63] "Unable to authenticate the request" err="[invalid bearer token, oidc: authenticator not initialized]"
-2022-05-26 13:46:11
-2022-05-26T11:46:11.200645694Z stderr F E0526 11:46:11.200494       1 authentication.go:63] "Unable to authenticate the request" err="[invalid bearer token, oidc: authenticator not initialized]"
-2022-05-26 13:46:10
-2022-05-26T11:46:10.282230799Z stderr F E0526 11:46:10.282080       1 oidc.go:224] oidc authenticator: initializing plugin: Get "https://<your-kkp.domain>/dex/.well-known/openid-configuration": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
-
-```
+Kubermatic tracks this as [kubermatic/kubermatic#12874](https://github.com/kubermatic/kubermatic/issues/12874).
 
+### Root Cause
 
-**Root cause**
+A bug in Cilium tracked as [cilium/cilium#12277](https://github.com/cilium/cilium/issues/12277) and [cilium/cilium#20550](https://github.com/cilium/cilium/issues/20550):
 
-The KKP [API Server network policy]({{< ref "../../tutorials-howtos/networking/apiserver-policies" >}}) is relying on the namespace label `kubernetes.io/metadata.name` which is automatically present on K8s 1.21 and above versions, but missing on K8s versions below 1.21. Due to the mismatch in the label selector, the access is denied.
+Kubernetes `NetworkPolicy` documentation [implies that allowing node access via CIDRs is supposed to work](https://kubernetes.io/docs/concepts/services-networking/network-policies/#what-you-can-t-do-with-network-policies-at-least-not-yet):
 
-**Solution**
+> Node specific policies (you can use CIDR notation for these, but you cannot target nodes by their Kubernetes identities specifically).
 
-As the issue is seen only with older versions of K8s which have reached end of life, the preferred solution is to upgrade the K8s to 1.21 or the latest version.
-In the case where upgrade is not desirable then a work around can be applied by adding a label to the `nginx-ingress-controller` namespace as shown below.
+But Cilium does not properly allow traffic even though `NetworkPolicies` with node CIDRs are in place.
 
-`kubectl label ns nginx-ingress-controller "kubernetes.io/metadata.name=nginx-ingress-controller"`
+### Solution
 
-### 2. Connectivity Issue in Pod-to-NodePort Service in Cilium + IPVS Proxy Mode
+At the moment (with KKP v2.24.0), there are two options available:
 
-**Problem**
+1. [Disable kube-apiserver NetworkPolicies in Seeds with Cilium as CNI]({{< ref "../../tutorials-howtos/networking/apiserver-policies/#in-a-seed-cluster" >}})
+2. Manually creating `CiliumNetworkPolicy` objects in each user cluster namespace to allow the erroneously traffic:
```

Comment:
```suggestion
2. Manually creating `CiliumNetworkPolicy` objects in each user cluster namespace to allow the erroneously discarded traffic:
```
nit: the original sentence seems to be missing something, maybe this reads better? ^^

Reply:
Fixed! thanks.